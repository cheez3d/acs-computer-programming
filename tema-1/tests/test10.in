Nine noble families fight for control over the mythical lands of Westeros
1266026622244433668033663369990777338#88777667777
Daenerys strikes back at her enemies
4
Daenerys Dragon
strikes smiles
her his
enemies friends
Can you picture Cersei starring in a comedy
