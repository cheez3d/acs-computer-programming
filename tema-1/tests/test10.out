1664446633066#66622555330333264445554443377770333444#4#44803336667770222666#66877766655506668883377708443306999844#444222#255505552663777706663330193377778337776667777
An ancient enemy returns
1377724666#66077776444555337777022#2#22255028044#44477770333777444336637777
8
1222266 99966688 744422288877733 122233777777733444 777782777777444664 44466 2 2226666333999 
777782777777444664 5
0
0
